docs/product-user-guide/src/main/asciidoc/business-central-settings-security-con.adoc

```diff
@@ -1,4 +1,4 @@
 [id='business-central-settings-security-proc']
-= Security settings
+= Security management
 
-To access the Security perspective, click *Roles*, *Groups*, or *Users* on the Settings perspective. Depending on which icon you click, the Security perspective opens on the *Roles*, *Groups*, or *Users* tab. Use roles and groups to manage privileges and settings for users. Any changes apply to all users with the affected role or group.
+You can perform security and permissions management tasks from the Security perspective. To access the Security perspective, click *Roles*, *Groups*, or *Users* on the Settings perspective. Depending on which icon you click, the Security perspective opens on the *Roles*, *Groups*, or *Users* tab. Use roles and groups to manage privileges and settings for users. Any changes apply to all users with the affected role or group.
```

Comment:
"perspective" => "page". As a rule of thumb never use the term perspective in a user oriented context.

Reply:
Changed as suggested.